Tests/Tests/RouterSpec.swift

```diff
@@ -279,6 +364,34 @@ class RouterSpec: QuickSpec {
                         expect(window.findTooltipView()).toEventuallyNot(beNil())
                     }
 
+                    it("will display a tooltip under the the campaign view") {
+                        let rootView = UIView()
+                        UIApplication.shared.keyWindow?.addSubview(rootView)
+                        router.accessibilityCompatibleDisplay = true
+                        let campaign = TestHelpers.generateCampaign(id: "test", type: .modal)
+                        router.displayCampaign(campaign, associatedImageData: nil, confirmation: true, completion: { _ in })
+                        expect(window.findIAMView()).toEventuallyNot(beNil())
+
+                        router.displayTooltip(tooltip,
+                                              targetView: rootView,
+                                              identifier: TooltipViewIdentifierMock,
+                                              imageBlob: imageBlob,
+                                              becameVisibleHandler: { _ in },
+                                              confirmation: true,
+                                              completion: { _ in })
+                        expect(window.findTooltipView()).toEventuallyNot(beNil())
+                        expect(window.findIAMView()).toEventuallyNot(beNil())
+                        let displayedTooltip = window.findTooltipView()!
+                        let displayedCampaign = window.findIAMView()!
+
+                        expect(displayedTooltip.superview).to(beIdenticalTo(displayedCampaign.superview))
+                        let tooltipIndex = displayedTooltip.superview?.subviews.firstIndex(of: displayedTooltip)
+                        let campaignIndex = displayedTooltip.superview?.subviews.firstIndex(of: displayedCampaign)
+                        expect(campaignIndex).to(beGreaterThan(tooltipIndex))
+                        window.findTooltipView()?.removeFromSuperview()
```

Comment:
Is this cleanup necessary?
Tooltip removal should be handled by line 320 and campaign view by line 50.

Reply:
Yes, it's because of the `rootView` as the targetView we need to remove it from the `rootView`.